invenio_i18n/views.py

```diff
@@ -53,11 +54,15 @@ def get_redirect_target():
             return target
 
 
-@blueprint.route('/<lang_code>/', methods=['POST'])
+@blueprint.route('/',  methods=['POST'])
+@blueprint.route('/<lang_code>/',  methods=['POST'])
 def set_lang(lang_code=None):
     """Set language in session and redirect."""
     # Check if language is available.
-    lang_code = lang_code.lower()
+    if lang_code is not None:
```

Comment:
@jirikuncar Ok for the ``or`` operator. But for the default value, if you are not setting any language in your request, ``lang_code`` is ``None``, so it is returning a 404 error. If you set a default value, it could have an awkward behavior because maybe the default babel is Czech, you are browsing the English version and you want to change to Spanish and it goes to Czech (Because Spanish is not available), I think this is not good.